modules/ROOT/pages/_partials/app-security/proc-calling-the-app-security-clientInit.adoc

```diff
@@ -0,0 +1,30 @@
+= Using the App Security Service
+
+Call the appSecurity clientInit function to implement the App Security Service. Call this service on application initialization.
+ 
+[tabs]
+====
+Cordova::
++
+--
+// end::excludeDownstream[]
+[source,javascript]
+----
+appSecurity.clientInit()
+.then(clientInit => {
+  /**  
+    You can use clientInit.data.disabled boolean 
+    to check if app has been disabled by admin.
+
+    The disabled message will be return if the app is disabled 
```

Comment:
```suggestion
    The disabled message is displayed if the app is disabled 
```

Reply:
Won't be displayed unless the developer/user implements it, I think returned is correct in this case

Reply:
gotcha, but also change to present tense, "the message is returned".. makes things easier in the long run